livre/eq/Channel.cpp

```diff
@@ -582,16 +624,16 @@ bool Channel::configExit()
 }
 
 void Channel::frameStart( const eq::uint128_t& frameID,
-                          const uint32_t frameNumber )
+                          const uint32_t frameCounter )
 {
-    _impl->_drawRange = eq::Range::ALL;
-    eq::Channel::frameStart( frameID, frameNumber );
+    _impl->frameStart( frameCounter );
+    eq::Channel::frameStart( frameID, frameCounter );
 }
 
 void Channel::frameDraw( const lunchbox::uint128_t& frameId )
 {
     eq::Channel::frameDraw( frameId );
-    _impl->frameDraw( frameId );
+    _impl->frameDraw();
```

Comment:
The fact that you call your impl function first in frameStart but here you call it last is a bit incoherent. I would call the impl methods preFrameStart and postFrameDraw to make it clear which is their position.